JogoDaVida/View.c

```diff
@@ -25,4 +25,20 @@ void ExibirGeracao(tipo_Individuo matriz[100][100], int tamanho){
 			printf("\t%d ", matriz[i][j].vivo);
 		}printf("\n");
 	}		
+
+
+void EntradaCoordenadas(int valores[y][x]){
+	int x,y;
+	printf("%s ");
+	scanf("%d", &x);	// pega o valor de x -- coluna
+	printf("%s "); 			// precisa de texto??
+	scanf("%d", &y);	// pega o valor de y -- linha 
+}
+bool EntradaBooleano(char valor[], char valorPositivo, char valorNegativo){
+	if (valor == "1"){
+		return valorPositivo;
+}
```

Comment:
Essa função é para entrada de valor booleano então ela retorna um "boolean".
Os parâmetros passados são: char valor[] : a mensagem que eu quero que seja printada para ao usuário, então a resposta do usuário será um char, se ele for igual ao parâmetro valorPositivo retorna TRUE se for igual ao parametro valorNegativo retorna FALSE, se não for nenhum dos dois, repita a perguntar até ter uma entrada adequada.